packages/dcos-sdk/README.md

```diff
@@ -0,0 +1,52 @@
+# DC/OS UI SDK
```

Comment:
This looks like a set of foundation services not like a software development kit, let’s please rename the package accordingly.

Reply:
I have issues with the word foundation. It is rather vague and doesn't provide any context. I'd say `react` is the foundation for our app, but not any of those extension providers / services.

[Wiki](https://en.wikipedia.org/wiki/Software_development_kit): A software development kit (SDK or devkit) is typically a set of software development tools that allows the creation of applications for a certain software package...

While I agree that the package doesn't include any tooling yet it still provides core services that enable plugin development. 

Coming from our vision having a shell app, sdk and a bunch of plugins I see this package as a foundation ;) for the SDK package.

Reply:
I'm open to naming suggestions

Reply:
including `plugin-baby`, `EventedContainer` etc. :)

Reply:
ping @orlandohohmeier